### PR TITLE
Added regPage code

### PR DESCRIPTION
**Pull request by vmjanani92**
Hi Karthik

Kindly review my code.

- Added RegPage
- Added RegPageTest

Regards,
VM Janani

### CODE REVIEW

File: src/Pages/RegPage.java [hunk 1]
```diff
@@ -0,0 +1,10 @@
+package Pages;
+
+public class RegPage {
+	
+		public static void doReg() {
+		
+		System.out.println("Reg Page");
```
Comment (unceasingwaves92) [inline src/Pages/RegPage.java:7]: Hello Janani,

Kindly add Regpage logic code.

Regards,
K. Karthikeyan

File: src/tests/RegPageTest.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package tests;
+
+public class RegPageTest {
+	
+	
+		public void RegPageTest() {
```
Comment (unceasingwaves92) [inline src/tests/RegPageTest.java:9]: Hello Janani,

Kindly add annotation test.

Regards,
K. Karthikeyan

**Review by unceasingwaves92 — CHANGES_REQUESTED** [review]
Hi Janani,

Please follow the comments and revert back to me.

Regards,
K. Karthikeyan

File: src/Pages/RegPage.java (hunk 1 above)
Comment (vmjanani92) [reply src/Pages/RegPage.java:7]: Hi Karthik,

Added logic code done.

Regards,
K. Karthikeyan

File: src/tests/RegPageTest.java (hunk 2 above)
Comment (vmjanani92) [reply src/tests/RegPageTest.java:9]: Hi Karthik,

Added testNG method.

Regards,
VM Janani

**Review by unceasingwaves92 — APPROVED** [review]
Hello Janani,

Well done changes. PR is approved.

Regards,
K. Karthikeyan

**Comment by vmjanani92** [comment]
Hi Karthik

Code is merged in Master branch.

Regards,
VM Janani

**Pull request merged**